### DISCUSSION TITLE
Why Is My Build Failing,How Can I Fix It?

### DISCUSSION THREAD

**Discussion by Rdx222OtherAccount** [html2app/discussions#15] [Q&A] [answered]
I have uploaded my zip with the structure shown in the video also there's a index.html pls check it.
[Clock.zip](https://github.com/user-attachments/files/17269161/Clock.zip)
Error Massage I am Getting- "06/10/2024, 16:52:48 (unknown/unknown) "
@yandeu @yandeu

**Comment by codingthat** [comment]
Same here, going straight to failing, no pending.

App is even simpler than last time, just an index.html file in www/ (and the assets in assets/, and config.json).

- 2024-12-03, 4:13:30 p.m. (unknown/unknown)
- 2024-12-03, 4:05:05 p.m. (unknown/unknown)
- 2024-12-03, 3:33:09 p.m. (unknown/unknown)
- 2024-12-03, 3:32:01 p.m. (unknown/unknown)


```
[
  {
    "buildId": "9333c543-8780-4c71-b34e-5daf2e16247b",
    "status": "FAILED",
    "time": "2024-12-03T15:13:30.135Z"
  },
  {
    "buildId": "5d2c2659-e2ca-401a-b713-575bae16db5a",
    "status": "FAILED",
    "time": "2024-12-03T15:05:05.130Z"
  },
  {
    "buildId": "ad489ba3-bff7-45a9-b244-a021f03bf6a4",
    "status": "FAILED",
    "time": "2024-12-03T14:33:09.916Z"
  },
  {
    "buildId": "51adb996-da49-4d65-8223-97400b5190fb",
    "status": "FAILED",
    "time": "2024-12-03T14:32:01.147Z"
  }
]
```

**Reply by yandeu** [reply]
Unfortunately, there is currently an issue with the pipeline.

**Reply by yandeu** [reply, marked as answer]
It should work again. But some things like fullscreen or orientation lock does not work yet.

**Reply by codingthat** [reply]
> It should work again. But some things like fullscreen or orientation lock does not work yet.

Working great now.  Thanks so much!  :pray:
